lib/commons/aria/index.js

```diff
@@ -376,8 +376,7 @@ lookupTables.role = {
 			one: ['rowgroup', 'row']
 		},
 		nameFrom: ['author'],
-		context: null,
-		implicit: ['table']
+		context: null
```

Comment:
I updated only the implicit roles who needed a update for this new rule to validate.

Reply:
What was the reason for removing the implicit table?

Reply:
https://www.w3.org/TR/html-aria/#table - An table element only has an implicit role of table following the current spec.